Makefile

```diff
@@ -300,8 +308,27 @@ test:
 		TEST_TOKEN_UNALLOWED="$(shell cat $(BUILD_FOLDER)/auth-tokenUnallowedString)" \
 		ENABLE_AUTH="true" \
 		WITH_AMS_SUBSCRIPTIONS="true" \
+		ENABLE_KUBE_API="false" \
 		go test -v ./subsystem/... -count=1 $(GINKGO_FOCUS_FLAG) -ginkgo.v -timeout 120m
 
+enable-kube-api-for-subsystem: $(BUILD_FOLDER) $(VERIFY_CLUSTER) generate-keys
+	skipper make deploy-service-requirements ENABLE_KUBE_API=true ENABLE_AUTH=false
+	$(call restart_service_pods)
+	$(MAKE) wait-for-service
+
+kube-api-tests:
```

Comment:
Let's make this target to call the regular test target and just override `FOCUS` and `ENABLE_KUBE_API`